### PR TITLE
docker: add Chinese translation

### PR DESCRIPTION
**Pull request by blueskyson**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/common/docker.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# docker
+
+> 管理 Docker 容器和镜像。
+> 更多信息: <https://docs.docker.com/engine/reference/commandline/cli/>.
+
+- 列出目前正在运行的 docker 容器:
```
Comment (marchersimon) [inline pages.zh/common/docker.md:6]: Those aren't full-width semicolons, right? I think that's what we agreed on in #5952

Comment (blueskyson) [reply pages.zh/common/docker.md:6]: Okay.

**Review by marchersimon — APPROVED** [review]
Thanks @blueskyson

File: pages.zh/common/docker.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps -a`
+
+- 透过镜像启动容器，并为容器命名：
+
+`docker run --name {{容器名称}} {{镜像}}`
+
+- 启动或停止现有容器：
+
+`docker {{start|stop}} {{容器名称}}`
```
Comment (einverne) [inline pages.zh/common/docker.md:20]: I think we can list more command here, such as `restart`

File: pages.zh_TW/common/docker.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps -a`
+
+- 透過映像檔啟動容器，並為容器命名：
+
+`docker run --name {{容器名稱}} {{映像檔}}`
+
+- 啟動或停止現有容器：
+
+`docker {{start|stop}} {{容器名稱}}`
```
Comment (einverne) [inline pages.zh_TW/common/docker.md:20]: Same here.

**Review by einverne — COMMENTED** [review]
Cool, thank you @blueskyson for the new pages.

File: pages.zh_TW/common/docker.md (hunk 3 above)
Comment (marchersimon) [reply pages.zh_TW/common/docker.md:20]: You commented twice on the same line 🤔

File: pages.zh/common/docker.md (hunk 2 above)
Comment (marchersimon) [reply pages.zh/common/docker.md:20]: Sounds good, but that should be done in a separate PR and applied to all translations.

File: pages.zh_TW/common/docker.md (hunk 3 above)
Comment (blueskyson) [reply pages.zh_TW/common/docker.md:20]: @marchersimon this is a **zh_TW** page, and this line looks the same as **zh** page by coincidence.

Comment (marchersimon) [reply pages.zh_TW/common/docker.md:20]: Oh, I just looked at the line number 😄

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**